paddle/fluid/operators/math/jit_kernel_blas.cc

```diff
@@ -14,10 +14,13 @@ limitations under the License. */
 
 #include "paddle/fluid/operators/math/jit_kernel.h"
 #include <string>
-#include "paddle/fluid/operators/math/jit_code.h"
 #include "paddle/fluid/operators/math/jit_kernel_macro.h"
 #include "paddle/fluid/platform/enforce.h"
 
+#ifdef PADDLE_WITH_XBYAK
```

Comment:
Who will define it?

Reply:
https://github.com/PaddlePaddle/Paddle/blob/8fc05e0373bb481e36b53c650f2dc00acf1b32a5/cmake/external/xbyak.cmake#L15-L19 

Let's why failed on Mac

Reply:
OK

Reply:
OK